Kickstarter-iOS/DataSources/BackerDashboardPagesDataSource.swift

```diff
@@ -0,0 +1,57 @@
+import KsApi
+import Library
+import UIKit
+
+internal final class BackerDashboardPagesDataSource: NSObject, UIPageViewControllerDataSource {
+  private let viewControllers: [UIViewController]
+
+  internal init(delegate: UIViewController, sort: DiscoveryParams.Sort) {
+    let backedController = BackerDashboardProjectsViewController()
+    let savedController = BackerDashboardProjectsViewController()
+
+    if let delegate = delegate as? BackerDashboardProjectsViewControllerDelegate {
+      backedController.configureWith(delegate: delegate, projectsType: .backed, sort: sort)
+      savedController.configureWith(delegate: delegate, projectsType: .saved, sort: sort)
+    }
+
+    var controllers: [UIViewController] = []
+    controllers[BackerDashboardTab.backed.rawValue] = backedController
+    controllers[BackerDashboardTab.saved.rawValue] = savedController
+    self.viewControllers = controllers
```

Comment:
just a note here... we don't have a concept of sections for our UIPageViewControllers, but I wanted to ensure there was a way to keep the order of pages (aka tabs) consistent. If you hate the word "tab" let me know.  So `BackerDashboardTab` represents a page section and this concept is used in place of `Int` everywhere a page is referenced in the code. This way, if we ever change the order of the enum, it will be changed everywhere that `BackerDashboardTab` is used. Probably a more elegant way to handle if I had the time to think more on it.